collections/ffs.go

```diff
@@ -0,0 +1,70 @@
+package collections
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type FFSInstance struct {
```

Comment:
So now we have a new Mongo collection: FFSInstance.
It contains all the information of a FFS instance which corresponds to a bucket.

Information here isn't available for the user (of course, it's in Mongo)... mostly, this is for tracking JobIDs, current status, and "ephemeral/temporal" things that we need to archiving.

Plus, of course, the auth token.

Reply:
Correct.